### PR TITLE
Updated

### LINKED ISSUE #5: Cherry/Orange card Internal spacing
For the cherry and orange cards, the spacing between the images and the text is great on mobile. 

It gets hard to see on larger sizes. Consider anchoring the text of each card to the bottom of the card, instead of the top.

Here's an example with some inline styles: 

Screenshot 2023-10-09 at 11 41 50 PM

The righthand size seems more pleasant to my eyes.

### LINKED ISSUE #6: Cover for egg and cup
Consider using `object-fit: cover` on the egg and cup for pleasantness.

e.g.:  

Screenshot 2023-10-09 at 11 45 17 PM

vs 

Screenshot 2023-10-09 at 11 45 28 PM

### PR DESCRIPTION
**Pull request by elidvenega**

### CODE REVIEW

**Comment by elidvenega** [comment]
I have worked on the issues @tomatrow

File: src/App.css [hunk 1]
```diff
@@ -111,47 +111,64 @@
...
 .content-img-cherry p {
   color: var(--green-color);
   text-align: center;
-  margin-top: 1rem;
   line-height: 1.5;
   padding: 0.3rem;
 }
 
+.content-img {
+  object-fit: cover;
```
Comment (tomatrow) [inline src/App.css:145]: hail

File: src/App.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ export default function App() {
       <OrangeHeader title="we are creatives" />
       <HamburgerMenu />
       <Content />
-      <Testimonials testimonials="client testimonials" />
+      <Testimonials subTitle="client testimonials" />
```
Comment (tomatrow) [inline src/App.tsx:14]: Nice, keeping it clean

File: src/components/Content.tsx [hunk 3]
```diff
@@ -36,21 +36,25 @@ export default function Content() {
           </div>
         </div>
         <div className="content-img-cherry">
-          <h2>Graphic Design</h2>
+        <div className="content-img-cherry-text">
+        <h2>Graphic Design</h2>
           <p>
             Great design makes you memorable. We deliver <br />
             artwork that underscores your brand message <br />
-            and captures potential clients attention.{" "}
```
Comment (tomatrow) [inline src/components/Content.tsx:43]: Hail prettier

**Review by tomatrow — APPROVED** [review]
Approval is mine

**Pull request merged**